cli/src/pcluster/resources/imagebuilder/parallelcluster_test.yaml

```diff
@@ -55,7 +55,7 @@ phases:
           commands:
             - |
               set -v
-              PATTERN=$(grep -F "default['cluster']['python-version']" {{ CookbookDefaultFile }})
+              PATTERN=$(jq '.default.cluster."python-version"' {{ CookbookDefaultFile }})
```

Comment:
Nitpick. Looks like we should change from `python-version` to `python_version`. By the way, how do we understand which config values are externally facing? Honestly, I would create (in a future PR) a json with only those values, so that it is clear what is exposed.